CHARTER.md

```diff
@@ -1,10 +1,11 @@
-# Open Container Initiative Charter v1.0
+# Open Container Initiative Charter v1.1
 
 Effective 13 November 2015
+Updated 6 May 2020
 
 ##  1. Mission of the Open Container Initiative (“OCI”).
 
-The Open Container Initiative provides an open source technical community within which industry participants may easily contribute to building a vendor-neutral, portable and open specification and runtime that deliver on the promise of containers as a source of application portability backed by a certification program.
+The Open Container Initiative provides an open source technical community within which industry participants may easily contribute to building a vendor-neutral, portable and open specifications and runtime that deliver on the promise of containers as a source of application portability backed by a certification program.
```

Comment:
"a [...] specifications" isn't grammatical.  I agree with changing this to plural to reflect the fact that we have multiple specifications, but we should keep the sentence grammatical as well.

```suggestion
The Open Container Initiative provides an open source technical community within which industry participants may easily contribute to building vendor-neutral, portable and open specifications and runtime that deliver on the promise of containers as a source of application portability backed by a certification program.
```

Should we change "runtime" to "reference implementations" or expand it to something like this?

```suggestion
The Open Container Initiative provides an open source technical community within which industry participants may easily contribute to building vendor-neutral, portable and open specifications, reference implementations, and tools that deliver on the promise of containers as a source of application portability backed by a certification program.
```